client/components/select-dropdown/index.jsx

```diff
@@ -154,6 +154,7 @@ class SelectDropdown extends Component {
 				const newChild = React.cloneElement( child, {
 					ref: ( child.type === DropdownItem ) ? 'item-' + refIndex : null,
 					key: 'item-' + index,
+					isDropdownOpen: this.state.isOpen,
```

Comment:
It would probably be fine to name is `isOpen`, the `Dropdown` part feels a bit redundant as it's used within the dropdown component.